recipe/meta.yaml

```diff
@@ -43,7 +44,8 @@ test:
   requires:
     - pip
   commands:
-    - pip check
+    - pip check || true   # [not win]
```

Comment:
Would it be appropriate to add a comment to indicate why these test results are disregarded?

Reply:
I wouldn't put a comment for this.
pip check may be flaky, as far what I've been told, and I haven't investigated yet what pip check does exactly and what makes it flaky
I'll have to get more information.

Reply:
Philosophically: if we don't care about the output of a test, should we even run the test?

Reply:
`pip check` inspects the environment (from a pip-perspective) and verifies the correct dependencies are installed. If the tests are being flaky or failing, I suggest that we fix the packaging issue rather than ignore the tests.

Reply:
I haven't seen the issue myself. I have just seen some PR being rejected for missing this check.
But I agree, a test is no good if its result is ignored. So I'll enable it again, and if it becomes an issue I will investigate.